tools/linter/golangci-lint/.golangci.yml

```diff
@@ -72,6 +72,10 @@ linters-settings:
       - default
       # Groups all imports with the specified Prefix.
       - prefix(github.com/envoyproxy/gateway)
+  copyloopvar:
```

Comment:
`copyloopvar` is not [enabled by default](https://golangci-lint.run/usage/linters#disabled-by-default), we should add it to `linters.enable`